packages/react-vapor/src/components/table-hoc/actions/TableHOCRowActions.ts

```diff
@@ -6,6 +6,7 @@ export const TableHOCRowActionsType = {
     select: 'SELECT_TABLE_ROW',
     deselectAll: 'DESELECT_ALL_TABLE_ROW',
     toggleCollapsible: 'TOGGLE_TABLE_COLLAPSIBLE_ROW',
+    closeAllToggleCollapsible: 'CLOSE_ALL_TOGGLE_TABLE_COLLAPSIBLE_ROW',

```

Comment:
Lets keep the name simple.

```suggestion
    closeAllRows: 'CLOSE_ALL_ROWS',
```

Reply:
`GAEL_DONT_LIKE_TO_READ_BIG_CHUNK_OF_TEXT_EXPECIALLY_WHEN_ITS_IN_ALL_CAPS`